client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -170,8 +170,12 @@
     CompletableFuture<Map<Stream, StreamCut>> generateStreamCuts(ScheduledExecutorService backgroundExecutor);
 
     /**
-     * Update Retention Stream-Cut for Streams in this Reader Group.
-     * See {#link ReaderGroupConfig.StreamDataRetention#CONSUMPTION_BASED_USER_STREAMCUT}
+     * Updates the stream-cuts above which this {@link ReaderGroup} would like data to be retained. The {@link StreamCut}s
```

Comment:
This comment doesn't read well. In particular, the first sentence is a bit confusing. Here is a potential re-write based on my understanding:

```
Updates the {@link StreamCut} objects to indicate the stream data that this reader group is releasing. Each stream cut is a position in the corresponding stream indicating that the data preceding this position can be truncated. The {@link StreamCut} objects provided to this API indicate that the reader group no longer needs the data preceding the stream cuts, possibly because the application has fully processed such data. Pravega, consequently, no longer needs
to retain data prior to those {@link StreamCut}s for this {@link ReaderGroup}.
```

Reply:
So explaining the stream-cut in the context of what we are trying to do here. I understand and will use this